tests/Makefile

```diff
@@ -14,11 +14,14 @@
 SHELL=/bin/bash
 TAG := dev
 
-.PHONY: help
+.PHONY: help install-test-driver
 help: ## Print this help
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: test
 
-test: ## Run all e2e tests
+test: install-test-driver ## Run all e2e tests
 	TAG=$(TAG) ./e2e.sh
+
+install-test-driver: ## Install e2e test driver
+	cd e2e-driver && go test ./... && go install
```

Comment:
Why you choose to use the test driver a binary instead of a library?

Reply:
How do we use the driver if it's library?